docs/root-application.md

```diff
@@ -24,24 +30,30 @@ does not have to. Note that if a child application is declared from within anoth
 will be maintained between the child applications. Instead, the child applications will be siblings and will be mounted
 and unmounted according to their own activity functions.
 
-In order to declare a child application, call the `declareChildApplication(path, activityFunction)` api. Example:
+In order to declare a child application, call the `declareChildApplication(name, howToLoad, activityFunction)` api. Example:
 
 ```js
 import { declareChildApplication } from 'single-spa';
 
-declareChildApplication("src/app1/app1.main.js", function() {
+declareChildApplication("app1", () => System.import("src/app1/main.js"), function() {
   return window.location.hash.indexOf("#/app1/") === 0;
 });
-declareChildApplication("src/app2/app2.main.js", function() {
+declareChildApplication("app2", () => System.import("src/app2/main.js"), function() {
   return window.location.hash.indexOf("#/app2/") === 0;
 });
 ```
 
 ### Path to child application
```

Comment:
Should this still be `Path to child application`? Or be `Child application name`? Along with this, it might be nice to somehow make each of these bullets correspond more to the function arguments above. I don't know if that means making the var names match more closely the headings?